code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -594,3 +594,12 @@
 	icon = 'icons/obj/clothing/clockwork_garb.dmi'
 	icon_state = "clockwork_cuirass_old"
 	armor = list("melee" = 5, "bullet" = 0, "laser" = -5, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 20)
+
+/obj/item/clothing/suit/ghost_sheet
+	name = "ghost sheet"
+	desc = "The hands float by themselves, so it's extra spooky."
+	icon_state = "ghost_sheet"
+	item_state = "ghost_sheet_item"

```

Comment:
For some reason this icon isn't displaying correctly, it's showing a series of boxes, like it's trying to display the entire .dmi at once or something. I have not the slightest idea why.